modules/lib/src/main/resources/assets/js/app/view/context/widget/version/VersionHistoryItem.ts

```diff
@@ -201,4 +206,171 @@ export class VersionHistoryItem {
     isPermissionsUpdated(): boolean {
         return this.status === VersionItemStatus.PERMISSIONS;
     }
+
+    getContentVersion(): ContentVersion {
+        return this.version;
+    }
+
+    isAlias(): boolean {
+        return !!this.alias;
+    }
+
+    getAlias(): VersionHistoryItemAlias {
+        return this.alias;
+    }
+
+    getAliasType(): AliasType {
+        if (!this.isAlias()) {
+            return null;
+        }
+
+        return this.getAlias().getType();
+    }
+
+    getAliasDisplayName(): string {
+        if (!this.isAlias()) {
+            return null;
+        }
+
+        return this.getAlias().getDisplayName();
+    }
+
+    createAlias(displayName: string, type: AliasType): VersionHistoryItem {
+        const versionAliasBuilder: Builder = this.clone();
+        const alias = new VersionHistoryItemAlias(displayName, type);
+        versionAliasBuilder.alias = alias;
+
+        return versionAliasBuilder.build();
+    }
+
+    clone(): Builder {
+        return new Builder(this);
+    }
+}
+
+export class Builder {
```

Comment:
Why do we need to export this class? If we do, then it shouldn't have such a generic name.